Content/Guides/shared/chunk/pe-para-chain-hostnames.htm

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<html>
+<html  xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd">
     <head>
         <meta name="description" content="" />
     </head>
-    <body>The <span class="Code">chain-hostnames()</span> option of syslog-ng can interfere with the way <MadCap:variable name="General.abbrev" xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd"></MadCap:variable> counts the log source hosts, causing syslog-ng to think there are more hosts logging to the central server, especially if the clients sends a hostname in the message that is different from its real hostname (as resolved from DNS). Disable the <b class="Code">chain-hostnames()</b> option on your log sources to avoid any problems related to license counting.</body>
+    <body>The <span class="Code">chain-hostnames()</span> option of syslog-ng can interfere with the way <MadCap:variable name="General.abbrev"></MadCap:variable> counts the log source hosts, causing syslog-ng to think there are more hosts logging to the central server, especially if the clients sends a hostname in the message that is different from its real hostname (as resolved from DNS). Disable the <b class="Code">chain-hostnames()</b> option on your log sources to avoid any problems related to license counting.</body>
```

Comment:
`The <span class="Code">chain-hostnames()</span> option can interfere with the way <MadCap:variable name="General.abbrev"></MadCap:variable> counts the log source hosts. As a result, <MadCap:variable name="General.abbrev"></MadCap:variable> falsely perceives several hosts logging to the central server`